modules/core/src/main/java/org/apache/ignite/internal/processors/security/IgniteSecurity.java

```diff
@@ -126,4 +127,33 @@ public default void authorize(SecurityPermission perm) throws SecurityException
      * false if it's used a default NoOp implementation.
      */
     public boolean enabled();
+
+    /**
+     * Creates user with the specified login and password.
+     *
+     * @param login Login of the user to be created.
+     * @param pwd User password.
+     * @throws IgniteCheckedException If error occurred.
+     */
+    @IgniteExperimental
```

Comment:
We don't need IgniteExperimental for internal classes.

Reply:
Reverted.